### PR TITLE
switch not the server naming

### PR DESCRIPTION
**Pull request by duritong**
The world moved on to the term puppetserver

### CODE REVIEW

**Comment by ekohl** [comment]
I don't see a backwards-incompatible label. @bastelfreak I thought the app was supposed to sync those.

**Comment by bastelfreak** [comment]
@ekohl added it. the app is currently in maintenance mode because the github api trolled me a few times. I hope to fix this next week.

File: manifests/rules/out/puppet.pp [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # manage outgoing puppet
 class nftables::rules::out::puppet (
-  Variant[String,Array[String,1]] $puppetmaster,
+  Variant[String,Array[String,1]] $puppetserver,
   Integer $puppetserver_port = 8140,
 ) {
-  any2array($puppetmaster).each |$index,$pm| {
+  any2array($puppetserver).each |$index,$ps| {
```
Comment (bastelfreak) [inline manifests/rules/out/puppet.pp:6]:
```suggestion
  [$puppetserver].flatten.each |$index,$ps| {
```

this should work as well and avoids the stdlib function call

**Review by bastelfreak — APPROVED** [review]

File: manifests/rules/out/puppet.pp [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # manage outgoing puppet
 class nftables::rules::out::puppet (
```
Comment (traylenator) [inline manifests/rules/out/puppet.pp:2]: There's no tests for this class, maybe be add some trivial ones.

**Review by traylenator — CHANGES_REQUESTED** [review]

File: manifests/rules/out/puppet.pp (hunk 2 above)
Comment (duritong) [reply manifests/rules/out/puppet.pp:2]: yeah wasn't sure, whether this should happen in this change

**Comment by traylenator** [comment]
The new test spec/acceptance/all_rules_spec.rb in #50 will need updating depending on if this or #50 gets there first - trivial.

**Comment by nbarrientos** [comment]
Needs rebasing to make the acceptance tests fail.

File: manifests/rules/out/puppet.pp (hunk 2 above)
Comment (duritong) [reply manifests/rules/out/puppet.pp:2]: added

**Review by traylenator — APPROVED** [review]

**Pull request merged**